confgenerator/testdata/valid/windows_metrics-no_conf/golden_otel.conf

```diff
@@ -0,0 +1,402 @@
+receivers:
+  prometheus/agent:
+    config:
+      scrape_configs:
+      - job_name: 'otel-collector'
+        scrape_interval: 1m
+        static_configs:
+        - targets: ['0.0.0.0:8888']
+  
+processors:
```

Comment:
I basically dump all processors required for the default otel windows to function. As long as it's not used by any pipelines, it's simply no-op. but worth asking @james-bebbington if defining the processor will cost any resources for void.

Reply:
This is noop and cost no resources if not used in pipelines.